src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql

```diff
@@ -111,7 +111,20 @@ VALUES
     ('0d592388-3d61-4f9d-9e11-6b0f5858acc0', 'patients_899k'       , 'patients'                     , 'per patient 899,000 patients and above'),
     ('2891ed7e-64bb-4ed6-8108-95c1fad9e2ee', 'patientUser'         , 'patient users'                , 'per patient user'),
     ('a10ae1df-b8a1-4f8d-aeb2-cd636e24d966', 'minuteUser'          , 'minutes/users'                , 'per minute per end user'),
-    ('8175143c-3350-4186-954c-d98172a7dfb0', 'gigabyte'            , 'gigabytes'                    , 'per gigabyte');
+    ('8175143c-3350-4186-954c-d98172a7dfb0', 'gigabyte'            , 'gigabytes'                    , 'per gigabyte'),
+    ('d9a7efe6-11a6-474a-99a1-03cd5909408c', 'pack_10k'            , 'packs'                        , 'Per Pack (10,000 mins/texts)'),
```

Comment:
`Per Pack` should be lower case, i.e. `per pack`, to be consistent.

Reply:
Updated pricing unit description to lower case.